core/shared/src/main/scala/zio/Chunk.scala

```diff
@@ -835,6 +835,7 @@ object Chunk {
     case x: Singleton[A]   => x.classTag
     case x: Slice[A]       => x.classTag
     case x: VectorChunk[A] => x.classTag
+    case _: BitChunk       => ClassTag.Boolean.asInstanceOf[ClassTag[A]]
```

Comment:
This is technically correct, I think. Although in any case where we look at this class tag and are tempted to use it (to create a new array), we should probably be special casing and instead creating a bit chunk... so hopefully we never use the class tag.